go.mod

```diff
@@ -415,6 +413,11 @@ require (
 
 require golang.org/x/exp v0.0.0-20221004215720-b9f4876ce741
 
+require (
+	gomodules.xyz/jsonpatch/v2 v2.2.0 // indirect
```

Comment:
Not sure why these got pulled out into a separate `require` block, but you can manually move them back.